### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] Table: Each of the cell value should be associated to its header.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11270]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
When the user access each of the cell values in the table , it is observed that only the first column cell value is not associated to its header and only the cell value is mentioned. Each of the cell value should be associated to its header.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use table and access each of the cell values in the table , it will observe that only the first column cell value is not associated to its header and only the cell value is mentioned by screen reader

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by nikolay-kolarov** [comment]
Hi @JyotiKumari1507,

I checked the html and the headers attribute is set also for the cells on the second row:
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/175b0788-1751-435f-a310-0bf2b4b336a3)
By principle when you enter in a column the header is announced once and if you navigate further in this column it is not repeated. But if you switch to another one, then it will be announced. And this works regardless of the row.

Could you please double check? If you still think something is missing, please give details on how to reach it.

Thanks,
Nikolay

**Comment by nikolay-kolarov** [comment]
Hi @JyotiKumari1507,

It is strange when you give another example now.
My tests on the page showed that the headers are announced. 
Could you please use the initial sample (https://sap.github.io/fundamental-ngx/#/platform/table/basic) and give concrete steps how to navigate in the table and where they are not read?

Thanks,
Nikolay

**Comment by nikolay-kolarov** [comment]
Also one more thing - the header is announced once you enter the column. If you then press arrow down it will not be announced (as you are in the same column). On the screenshot I see the extract of the announcement for the second row. So maybe if you entered on the first row, the header was announced and then on arrow down it wasn`t. But as described this is coming from the screen reader as optimization.

**Comment by JyotiKumari1507** [comment]
@nikolay-kolarov @droshev As per accessibility team first column of first row is also not associated with its hearder, rest column of first row was associated with header. you can refer  this screen shot. If required i can ask recording from accessibility team 
![image](https://github.com/SAP/fundamental-ngx/assets/157268666/bacbf0ef-9372-46c9-b1dc-69332cc012cb)

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

This looks more like a list, but I do not see such multicolumn like example in the storybook, so that I can test it.
Could you please clarify what is the used control with Jyoti and give me an example where I can test?

Thanks,
Nikolay

**Comment by droshev** [comment]
@JyotiKumari1507 Which component did you use in this scenario?

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev can you pls let me know what input field expecting from us?

**Comment by droshev** [comment]
@JyotiKumari1507 can you create a stackblitz example, write the steps and what the expectation is?

**Comment by droshev** [comment]
@JyotiKumari1507 Can you create a stackblitz example, steps to reproduce and what the expectation is? otherwise we will close the ticket.

**Comment by divyamalhotra488** [comment]
Hi @droshev, you can try with the example [here](https://stackblitz.com/edit/prywkj?file=package.json)

**Comment by droshev** [comment]
@nikolay-kolarov Do you think you can check the last example and validate whether the announced data is not enought? thank you in advance.

**Comment by nikolay-kolarov** [comment]
Hello @divyamalhotra488,
Hi @droshev,

I do not find an issue. The column headers are announced when you enter the table - you will go either on the column header and then move below, or if you were already in the table and come back then the header is still announced. Also when you switch between the columns they are announced.
Please check this recording:

https://github.com/user-attachments/assets/c6d41389-febe-4856-a81f-2432140207f0

Also the headers are properly related:
![image](https://github.com/user-attachments/assets/adccf24d-e76b-4aaa-a016-8237c498f273)

So from my perspective it works as expected and has the needed connection.
Please provide a recording or explain in which pattern, interaction steps, you expect the announcement.

Thanks,
Nikolay

**Comment by MeghanaVernekar** [comment]
Hello Colleagues,

Retested the issue and the issue is resolved. Hence, we can close the issue.

Regards,
Meghana.
